.husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
Does this script run locally? I'm on Windows, and this wouldn't run.

Reply:
😱  That's a very good point! Let me look into that!

Reply:
OK, I think we are good with Windows:
From Husky docs:

> [Does it work on Windows?](https://typicode.github.io/husky/#/?id=does-it-work-on-windows)
> Yes. When you install Git on Windows, it comes with the necessary software to run shell scripts.